### DISCUSSION TITLE
Default views missing from installation

### DISCUSSION THREAD

**Discussion by treadnought** [statamic/cms#6958] [Troubleshooting] [answered]
I'm trying to install Statamic into a new Laravel app in a WSL2 Sail setup.
`sail composer require statamic/cms --with-dependencies` seems to work, but the antlers views are not present in the views folder, just the default Laravel welcome blade.
Is there something I've missed?

**Comment by duncanmcclean** [comment, marked as answer]
I believe the default Antlers views are only there if you created your project with the Statamic CLI as apposed to installing in an existing Laravel app (which you've done by `composer require`ing it)
